java/dataflow-bulk-delete/README.md

```diff
@@ -0,0 +1,23 @@
+# Bulk Delete
+
+## An example of how to delete a list of row prefixes.
+
+This demonstrates how to delete a lot of data to workaround the rate limiting of DropRowRanges. 
+There are 2 examples provided simple and advanced. Both accomplish the same thing.
+
+## Simple
+
+This is a bare bones example to show the basic idea of taking a list of row prefixes and
+scanning Bigtable for matching rows and deleting those row in bulk.
+
+
+## Advanced
```

Comment:
Can we please get rid of Advanced?  It's too complex for what we're trying to achieve.

Reply:
Done